charts/csm-authorization/templates/proxy-server.yaml

```diff
@@ -54,7 +54,7 @@ rules:
     resources: ["events"]
     verbs: ["watch"]
   - apiGroups: ["csm-authorization.storage.dell.com"]
-    resources: ["storages"]
+    resources: ["storages","csmtenants"]
```

Comment:
is a space needed?

Reply:
I think it is not but since it used in other places, I have added space